<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CurrentDataGps</class>
 <widget class="QWidget" name="CurrentDataGps">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Палетка</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>39</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>3</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>1024</width>
       <height>600</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>1920</width>
       <height>1080</height>
      </size>
     </property>
     <widget class="QPushButton" name="cancelButton">
      <property name="geometry">
       <rect>
        <x>840</x>
        <y>39</y>
        <width>108</width>
        <height>100</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"> QPushButton:hover:pressed {
    padding-right: 15px; /* make room for the arrows */
	border-image: url(:/Pyramid/close_tool_n.png);
     border-width: 3;}
 
QPushButton{
     padding-right: 15px; /* make room for the arrows */
   	border-image: url(:/Pyramid/close_tool.png);
     border-width: 3;}

 QPushButton:hover {
    padding-right: 15px; /* make room for the arrows */
    	border-image: url(:/Pyramid/close_tool.png);
     border-width: 3;}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>53</y>
        <width>391</width>
        <height>71</height>
       </rect>
      </property>
      <property name="palette">
       <palette>
        <active>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Text">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="ButtonText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Text">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="ButtonText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>120</red>
            <green>120</green>
            <blue>120</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Text">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>120</red>
            <green>120</green>
            <blue>120</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="ButtonText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>120</red>
            <green>120</green>
            <blue>120</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
      <property name="font">
       <font>
        <family>Trebuchet MS</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>текущие данные GPS/Глонасс</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="background">
      <property name="geometry">
       <rect>
        <x>31</x>
        <y>30</y>
        <width>941</width>
        <height>531</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">padding-right: 15px; /* make room for the arrows */
border-image: url(:/Pyramid/fon_next_menu.png);
border-width: 3;
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QComboBox" name="typeMessageCB">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>70</y>
        <width>141</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Trebuchet MS</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <item>
       <property name="text">
        <string>$ALL</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>$GPGGA</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>$GPRMC</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>$GPGSA</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>$GPGSV</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>$GPGLL</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>$GPVTG</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>$GNZDA</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>$GNVTG</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>$GNRMC</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>$GNGSA</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>$GNGLL</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>$GNGGA</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="pauseBut">
      <property name="geometry">
       <rect>
        <x>710</x>
        <y>40</y>
        <width>101</width>
        <height>100</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"> QPushButton:hover:pressed {
    padding-right: 15px; /* make room for the arrows */
	border-image: url(:/Pyramid/send_otv_n.png);
     border-width: 3;}
 
QPushButton{
     padding-right: 15px; /* make room for the arrows */
   	border-image: url(:/Pyramid/send_otv.png);
     border-width: 3;}



 QPushButton:hover:checked {
    padding-right: 15px; /* make room for the arrows */
	border-image: url(:/Pyramid/send_otv_n.png);
     border-width: 3;}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="curr">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>150</y>
        <width>861</width>
        <height>381</height>
       </rect>
      </property>
      <property name="palette">
       <palette>
        <active>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>216</red>
            <green>216</green>
            <blue>216</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>216</red>
            <green>216</green>
            <blue>216</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>240</red>
            <green>240</green>
            <blue>240</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
      <property name="font">
       <font>
        <family>Trebuchet MS</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
      <property name="plainText">
       <string/>
      </property>
      <property name="overwriteMode">
       <bool>false</bool>
      </property>
      <property name="centerOnScroll">
       <bool>false</bool>
      </property>
     </widget>
     <zorder>background</zorder>
     <zorder>cancelButton</zorder>
     <zorder>label</zorder>
     <zorder>pauseBut</zorder>
     <zorder>curr</zorder>
     <zorder>typeMessageCB</zorder>
    </widget>
   </item>
   <item row="1" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>4</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../Resources/PyramidWidgets.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>CurrentDataGps</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>893</x>
     <y>88</y>
    </hint>
    <hint type="destinationlabel">
     <x>511</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
